File: services/mr-viewer-service/src/main/java/com/researchex/mrviewer/api/ServiceMetadataResponse.java
package com.researchex.mrviewer.api;

import java.time.OffsetDateTime;

/** MR Viewer 서비스가 노출하는 기본 메타데이터 응답. */
public record ServiceMetadataResponse(
    String serviceName, String description, OffsetDateTime timestamp) {}

File: services/mr-viewer-service/src/main/kotlin/com/researchex/mrviewer/api/ServiceMetadataResponse.kt
package com.researchex.mrviewer.api

import java.time.OffsetDateTime

/** MR Viewer 서비스가 노출하는 기본 메타데이터 응답이다. */
data class ServiceMetadataResponse(
    val serviceName: String,
    val description: String,
    val timestamp: OffsetDateTime,
)
